src/RestApiClient.js

```diff
@@ -80,6 +80,8 @@ class RestApiClient {
 
     if (query && typeof query === 'object') {
       url.search = stringifyQuery(query);
+    } else {
+      url.search = query;
```

Comment:
Instead of using `if/else` might be cleaner to only modify the query if an object, but the end result is to always save `query` to the `url.search` property:

```js
if (query && typeof query === 'object') {
  query = stringifyQuery(query);
}

url.search = query;
```

Reply:
oh much better! fixed :)

Reply:
Oh sorry I misunderstood the earlier pr, I thought it was saying the `url` already had a query in it, and this was breaking it?